rollup.config.js

```diff
@@ -1,16 +1,18 @@
-import pkg from './package.json';
-import commonjs from 'rollup-plugin-commonjs';
-import nodeBuiltins from 'rollup-plugin-node-builtins';
-import nodeResolve from 'rollup-plugin-node-resolve';
-import babel from 'rollup-plugin-babel';
+'use strict';
+
+const pkg = require('./package.json');
+const commonjs = require('rollup-plugin-commonjs');
+const nodeBuiltins = require('rollup-plugin-node-builtins');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const babel = require('rollup-plugin-babel');
 
 const input = 'index.js';
 const plugins = [
+  nodeBuiltins(),
```

Comment:
Just a reminder to remove these since they are only needed in karma

Reply:
Just wanted to confirm, both nodeGlobals and nodeBuiltins are karma-only? We hesitate because we see nodeBuiltins in the rollup config on master at the moment

Reply:
Ah yes. Builtins should be lower, my bad. nodeGlobals isn't needed and you don't have it here anyway so that's good. This file should be unmodified from master and by assocation so should dist/bson.bundle.js.

Reply:
Thanks for clarifying, will re-order to match master!